components/NavLink/NavLink.tsx

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React, { JSX } from 'react';
```

Comment:
Need a unit test for this new component.

Reply:
Ah so I should take the relevant tests form the original nav.tsx and move them into a new NavList.test file? That makes sense.

Reply:
No. You should create a brand new NavLink unit test file. It should ensure that all the props that can change will change correctly.

Reply:
@shashilo Latest changes address this _i think_